metabase/metabase
# Reserve component height with defaults from visualization type in embedded questions

Closes https://github.com/metabase/metabase/issues/42199

### Description

The embedded component currently does not render unless the component's parent is a fixed-height flexbox (see the above issue), and the user cannot set the height. This PR defines a fixed default height based on [the Figma spec](https://www.figma.com/file/JgPUvIfWeNzTiA0PL4QeUl/Viz-card-default-and-min-sizes?type=design&node-id=240%3A5607&mode=design&t=DR5wJ4aS3JqZrKrs-1), and allows user to override the default via the `height` prop.

### How to verify

- Checkout the `remove-height-workaround` branch in the demo app
- Navigate to `/admin/internal/kitchen-sink` in the demo. In this branch, the kitchen sink does not explicitly set the component height for the charts, so it falls back to the default value.
- Add the `height` prop, and the height should be customizable.

### Demo

![CleanShot 2567-05-07 at 23 19 28@2x](https://github.com/metabase/metabase/assets/4714175/90d9e081-f8be-4b8a-8fa1-1d230586d669)
